### DISCUSSION TITLE
Future of this repository

### DISCUSSION THREAD

**Discussion by jritta** [peridigm/peridigm#291] [General]
Dear community,

I'm a little frustrated because each pull request takes a long time before a decision is made. So I'm wondering if we will continue to develop Peridigm on GitHub or if this repository will die out?

I would like to contribute other stuff I implemented, but if there is no interest in further developing this repository, I will say goodbye to the community here and only develop in-house in future.

Best regards,
Jonas

**Comment by johntfoster** [comment]
@jritta I think we'd like to keep this thing alive, it's just that all of the original developers have moved on to management roles, etc. and don't have the time for day-to-day maintenance.  Would you be interested in being a maintainer?

**Reply by oldninja** [reply]
I am interested and would like to collaborate with @jritta on this.

**Reply by jritta** [reply]
Dear @johntfoster,

I fully understand this point and that we all cannot respond immediately to a request.

Thank you very much for the offer. I feel honored!  In principle I am interested in it. Could you maybe briefly mention what the expectations would be for me? (if requested as a PM I can also email you)

Best,
Jonas

**Reply by jritta** [reply]
Dear maintainers,

Was there already a decision made or is something planned?

Best,
Jonas

**Reply by jritta** [reply]
Dear community,

I would like to ask again if we made any progress in this regard?

Best,
Jonas
